github/with_appengine.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2017 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// This file provides glue for making github work on App Engine.
+
+// +build appengine
```

Comment:
Minor and has no effect, but I'd suggest swapping the order of these two lines, like so:

```
// +build appengine

// This file provides glue for making github work on App Engine.
```

This is because I think the build constraints have higher priority and should be closer to top of the file. It's better for people to see the constraints as soon as possible, before the rest of the file. The only exception is that we place the copyright comment on top of that.

Somewhat of a precedent is [here](https://github.com/google/go-github/blob/04b64a7f8ccd7701ed660231f3805bd585fb486c/github/gen-accessors.go#L6-L8).